tinker_cookbook/renderers/gpt_oss.py

```diff
@@ -1,40 +1,193 @@
 """
-GptOssRenderer - OpenAI's open source model format.
+GptOssRenderer - OpenAI's open source model format (Harmony).
 
 Format like this (no newlines between messages, last message should end with <|return|> but
 be replaced by <|end|> when continuing the convo):
     <|start|>system<|message|>You are ChatGPT...<|end|><|start|>user<|message|>How much is 1+1?<|end|><|start|>assistant<|channel|>final<|message|>2<|end|><|start|>
+
+Harmony channels:
```

Comment:
please say a bit more about channels -- one per assistant message? what is the channel intended to control?